autoload/lsp/lspserver.vim

```diff
@@ -885,18 +885,6 @@ def GotoSymbolLoc(lspserver: dict<any>, msg: string, peekSymbol: bool,
 		  cmdmods: string)
   var reply = lspserver.rpc(msg, GetLspTextDocPosition(true))
   if reply->empty() || reply.result->empty()
-    var emsg: string
-    if msg ==# 'textDocument/declaration'
-      emsg = 'Error: symbol declaration is not found'
-    elseif msg ==# 'textDocument/typeDefinition'
-      emsg = 'Error: symbol type definition is not found'
-    elseif msg ==# 'textDocument/implementation'
-      emsg = 'Error: symbol implementation is not found'
-    else
-      emsg = 'Error: symbol definition is not found'
-    endif
-
-    util.WarnMsg(emsg)
```

Comment:
When trying to jump to an non-existing symbol declaration/definition, we do want to display a warning message.  If this is removed, then how will the user know that the symbol is not found?

Reply:
for example: `Error(LSP): request textDocument/implementation failed (no object found, code = 0)`

Reply:
I don't see this error message with clangd when trying to jump to a non-existing symbol definition.  Which language server are you using?

Reply:
gopls for golang. supposed this should be standard later or now?

Reply:
pls check, i push a new commit, now should be able to handle both clangd and gopls.